### PR TITLE
Fixed codeforces module

### LINKED ISSUE #751: Codeforces module isn't working properly.
The code-forces module isn't working properly and throwing errors on a run. 

You can locate the files at `/src/scrape-up/codeforces/user.py`

### PR DESCRIPTION
**Pull request by lazyMonk1010**
Refactored the user_get_data method in USER class of codeforces module to correctly fetch the user data from codeforces API. The previous approach had issues with endpoint and the way it is retrieving the data , which should now be resolved with this change.

Fixes #751

## Description

> The changes made in this pull request should be briefly described.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [ ] Have you adhered to the repository's defined coding convention rules?
- [ ] Have you updated the 'documentation.md' file with the method/function documentation?
- [ ] Have you sent a message along with the result or response?
- [ ] Have you used the try-catch technique?
- [ ] Has the method/class been added to the documentation (md file)?

## Screenshots

> Uploading a screenshot illustrating the approach you developed for validation is required.

## Additional Notes/Comments

> Any additional remarks or suggestions concerning this pull request are welcome.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/codeforces/user.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ class USER:
     Fetches user data from CodeForces Competitive Platform
     """
 
-    user_url = "https://codeforces.com/profile/{}"
+    api_base_url = "https://codeforces.com/api/"
```
Comment (nikhil25803) [inline src/scrape_up/codeforces/user.py:10]: Do not use APIs @lazyMonk1010 |
We are building an API alternative, so it doesn't make any sense to use API under the hood.
Check if you can fix this module using any scraping library(not selenium or chrome web driver).

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

**Comment by lazyMonk1010** [comment]
Okay I will fix this module

**Comment by lazyMonk1010** [comment]
Hey @nikhil25803 I made the changes you asked and used scraping library instead of API

**Comment by lazyMonk1010** [comment]
hey @nikhil25803  can you check and merge the pr ,

**Comment by nikhil25803** [comment]
Sure, will check it by EOD @lazyMonk1010. 
Sorry, was busy in something.

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great work @lazyMonk1010  !!
Do not forget to
+ Share your contribution on socials(twitter mainly), tag us and Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Comment by lazyMonk1010** [comment]
Thank you @nikhil25803 for merge, glad to contribute. I'll be sure to share it on socials

**Comment by lazyMonk1010** [comment]
Hey nikhil! Thankyou so much for merging my pr .
I want to know clueless and your Twitter username.

**Comment by nikhil25803** [comment]
My username -  humans_write
Clueless - by_clueless

**Pull request merged**